src/PowerShellEditorServices/Services/Symbols/PesterDocumentSymbolProvider.cs

```diff
@@ -59,8 +59,10 @@ private static bool IsPesterCommand(CommandAst commandAst)
                 return false;
             }
 
-            // Ensure the first word is a Pester keyword
-            if (!PesterSymbolReference.PesterKeywords.ContainsKey(commandAst.GetCommandName()))
+            // Ensure the first word is a Pester keyword and in Pester-module if using module-qualified call
+            string commandName = CommandHelpers.StripModuleQualification(commandAst.GetCommandName(), out ReadOnlyMemory<char> module);
+            if (!PesterSymbolReference.PesterKeywords.ContainsKey(commandName) ||
+                (!module.IsEmpty && !string.Equals(module.ToString(), "pester", StringComparison.OrdinalIgnoreCase)))
```

Comment:
More efficient way to compare the module? Unfamiliar with the Memory/Span types

Reply:
Yep!

```suggestion
                module.Span.Equals("pester".AsSpan(), StringComparison.OrdinalIgnoreCase)
```

Reply:
I asked ChatGPT and while this is the most straightforward way to do it, it can be done more efficiently like so:
```c#
module.Span.SequenceEqual("pester".AsSpan())
```
And I guess the `"pester"` string could be made as a span itself.

As to _why_ that `out` is a ROM<char> it was because:

> Storing moduleName as ROMemory saves a string allocation in the common case where it is not needed.

/cc @SeeminglyScience

Reply:
Thanks! 🙂